packages/dm-core/src/hooks/useDocument.tsx

```diff
@@ -67,6 +69,9 @@ export function useDocument<T>(
       })
       .catch((error: AxiosError<ErrorResponse>) => {
         console.error(error)
+        toast.error(
+          'Unable to retrieve document, with message: ' + error.message
```

Comment:
```suggestion
          'Unable to retrieve document, with message: ' + error.response?.data.message ?? error.message
```

to print the message returned by the API, this expression can be used 🙌

![image](https://github.com/equinor/dm-core-packages/assets/69512295/e6aa799c-c07d-40f4-af01-0150278f3677)